docs/src/testing/using-cheatcodes.md

```diff
@@ -0,0 +1,158 @@
+# Using Cheatcodes
+
+When testing smart contracts, often there are parts of code that are dependent on specific blockchain state or are
+triggered on specific transaction properties.
```

Comment:
This line is a bit confusing, we could skip it I think

Reply:
I'm not so sure. In my opinion, it justifies the cheatcodes existance